build.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/build_config.h b/build_config.h
```

Comment:
Put the patch file in a separate directory like `patch/`, and also remember to update instructions in https://github.com/v8-riscv/v8/wiki/get-the-source.

Reply:
Fixed. I will update the wiki once it is merged.